release-please-config.json

```diff
@@ -0,0 +1,17 @@
+{
+    "release-type": "go",
+    "bump-minor-pre-major": true,
+    "bump-patch-for-minor-pre-major": true,
+    "include-component-in-tag": false,
```

Comment:
@rkodev we need the "include-v-in-tag" to be set to true, sorry for not caching that earlier.
https://github.com/googleapis/release-please/blob/656b9a9ad1ec77853d16ae1f40e63c4da1e12f0f/schemas/config.json#L81
https://github.com/microsoftgraph/msgraph-sdk-php/pull/1532/files#diff-c55d4dcb68c348b2c06d263819702fbce72eeeb35b662956d02049a5a18fcf2bR13